README.md

```diff
@@ -30,6 +30,8 @@ Spectator helps you get rid of all the boilerplate grunt work, leaving you with
 
 ## Table of Contents
 
+- [Features](#features)
```

Comment:
Remove Features and Table of Contents, please

Reply:
Thank you, it was added automatically by extension